core/core_test.go

```diff
@@ -105,46 +90,6 @@ func TestCertificateRequest(t *testing.T) {
 	}
 }
 
-func TestMergeChallenge(t *testing.T) {
-	tls := true
-	t1 := time.Now()
-	t2 := time.Now().Add(-5 * time.Hour)
-	challenge := Challenge{
-		Type:      ChallengeTypeSimpleHTTP,
-		Status:    StatusPending,
-		Validated: &t1,
-		Token:     "asdf",
-	}
-	response := Challenge{
-		Type:      ChallengeTypeSimpleHTTP,
-		Status:    StatusValid,
-		Validated: &t2,
-		Token:     "qwer",
-		TLS:       &tls,
-	}
-	merged := Challenge{
-		Type:      ChallengeTypeSimpleHTTP,
-		Status:    StatusPending,
-		Validated: &t1,
-		Token:     "asdf",
-		TLS:       &tls,
-	}
-
-	probe := challenge.MergeResponse(response)
```

Comment:
Since MergeResponse became just one line with an unhelpful switch around it, I've deleted the single use of it in the ra and replaced it with:

	authz.Challenges[challengeIndex].KeyAuthorization = response.KeyAuthorization